frontend/catalyst/api_extensions/quantum_operators.py

```diff
@@ -387,19 +388,23 @@ def _callee():
 
 
 ## IMPL ##
-class MidCircuitMeasure(HybridOp):
+class MidCircuitMeasure(HybridOp, MidMeasureMP):
     """Operation representing a mid-circuit measurement."""
 
     binder = qmeasure_p.bind
 
+    def __init__(self, in_classical_tracers, out_classical_tracers, regions: List[HybridOpRegion]):
+        HybridOp.__init__(self, in_classical_tracers, out_classical_tracers, regions)
+        # Initializing MidMeasureMP attributes
+        self.wires = [in_classical_tracers[0]]
+        self.postselect = in_classical_tracers[1]
+        # TODO: Check if we need to initialize other MidMeasureMP attributes as well
```

Comment:
Should we call `MidMeasureMP.__init__` or no?

Reply:
I was struggling with that, but the only attributes we really need to care about are the wires and `postselect`, and `MeasurementProcess.__init__` may have validation we want to skip, hence why I didn't call it.

Reply:
May be worth trying out, if it is successful the operations will be a little closer already 😅